aim/db/migration/alembic_migrations/versions/f18e545de625_bgp_peer_prefix.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2018 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""bgp peer prefix
+
+Revision ID: f18e545de625
+Revises: 310941aa5ee1
+Create Date: 2018-01-21 22:19:15.594994
+
+"""
+# revision identifiers, used by Alembic.
+revision = 'f18e545de625'
+down_revision = '310941aa5ee1'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.mysql import VARCHAR
+
+
+def upgrade():
+    op.add_column('aim_l3outsides', sa.Column('bgp_enable', sa.Boolean))
```

Comment:
server default needs to be set to False, or existing L3Outs will have null value (or even worse, True) after system upgrade

Reply:
I will make this change, I thought bgp_enable = False in the resource init ensures that existing l3outs will not be 'bgp_enabled". bgp_enable =None will also result in the same..